app/models/service_template_transformation_plan_task.rb

```diff
@@ -52,13 +51,12 @@ def task_active
   # This method returns true if all mappings are ok. It also preload
   #  virtv2v_disks and network_mappings in task options
   def preflight_check
+    raise 'OSP destination and source power_state is off' if destination_ems.emstype == 'openstack' && source.power_state == 'off'
+    update_options(:source_vm_power_state => source.power_state) # This will determine power_state of destination_vm
```

Comment:
this block is the essence of automate method: assesstransformation and its effect
https://github.com/ManageIQ/manageiq-content/blob/a9421bc26960fae1a78d335c6ab4bee28f948fbe/content/automate/ManageIQ/Transformation/Common.class/__methods__/assesstransformation.rb#L15

The following 2 lines are the spirit of the corresponding code in the Automate, essentially cancel out each other's effect and so are dropped
Refer to https://github.com/ManageIQ/manageiq-content/blob/a9421bc26960fae1a78d335c6ab4bee28f948fbe/content/automate/ManageIQ/Transformation/Infrastructure/VM/Common.class/__methods__/poweroff.rb#L16
   ```
 update_options(:power_off => true)
  raise 'source power_state is on and options[:power_off] is false' if source.power_state == 'on' && 
!options[:power_off]
```